crates/biome_js_formatter/tests/prettier_tests.rs

```diff
@@ -31,12 +31,30 @@ fn test_snapshot(input: &'static str, _: &str, _: &str, _: &str) {
         }
     };
 
+    if is_sloppy_mode_case(test_file.relative_file_name()) {
+        source_type = source_type.with_module_kind(ModuleKind::Script)
+    }
+
     let options = JsFormatOptions::new(source_type)
         .with_indent_style(IndentStyle::Space)
         .with_indent_width(2.into());
+
     let language = language::JsTestFormatLanguage::new(source_type);
 
     let snapshot = PrettierSnapshot::new(test_file, language, options);
 
     snapshot.test()
 }
+
```

Comment:
We could add a link to the [MDN docs](https://developer.mozilla.org/en-US/docs/Glossary/Sloppy_mode) about the sloppy mode. I didn't know that sloppy mode is script/non-strict mode.

Reply:
@Conaclos Maybe is better to change it to is_non_strict_mode?

Reply:
I let you choose. Otherwise, feel free to merge :)